source/integrate-with-integration-environment/process-identity-information.html.md.erb

```diff
@@ -51,11 +51,22 @@ Content-Type: application/json
       "addressCountry": "GB",
       "validUntil": "2022-01-01"
     }
+  ]
+},
+  "https://vocab.account.gov.uk/v1/drivingPermit": [
+    {
+      "expiryDate": "2023-01-18",
+      "issueNumber": "5",
+      "issuedBy": "DVLA",
+      "fullAddress": "122 BURNS CRESCENT EDINBURGH EH1 9GP",
+      "personalNumber": "DOE99802085J99FG",
+      "issueDate": "2010-01-18"
```

Comment:
See above comment - the `issueDate` property is no longer being added to the drivingPermit claim